src/components/TabSelect/example.js

```diff
@@ -6,18 +6,33 @@ import Tab from './Tab';
 
 export default function TabSelectExample() {
   return (
-    <Example>
-      <TabSelect>
-        <Tab label="A" name="a">
-          Tab A
-        </Tab>
-        <Tab label="B" name="b">
-          Tab B
-        </Tab>
-        <Tab label="C" name="c">
-          Tab C
-        </Tab>
-      </TabSelect>
-    </Example>
+    <div>
+      <Example>
+        <TabSelect>
+          <Tab label="A" name="a">
+            Tab A
+          </Tab>
+          <Tab label="B" name="b">
+            Tab B
+          </Tab>
+          <Tab label="C" name="c">
+            Tab C
+          </Tab>
+        </TabSelect>
+      </Example>
+      <Example>
```

Comment:
Now that we have two examples here, maybe you can add `<Info>` titles above each example as in e.g. Time Travel:

![image](https://user-images.githubusercontent.com/1216874/41232517-ebe4270c-6d86-11e8-90dc-e899eb48cfb2.png)

Reply:
👌